spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/KafkaMessageChannelBinder.java

```diff
@@ -542,62 +419,53 @@ public Object doWithRetry(RetryContext context) throws RuntimeException {
 		}
 	}
 
-	private Binding<MessageChannel> createKafkaConsumer(String name, final MessageChannel moduleInputChannel, Properties properties,
-			String group, long referencePoint) {
-
-		validateConsumerProperties(groupedName(name, group), properties, SUPPORTED_CONSUMER_PROPERTIES);
-		KafkaPropertiesAccessor accessor = new KafkaPropertiesAccessor(properties);
-
-		int maxConcurrency = accessor.getConcurrency(defaultConcurrency);
+	private Binding<MessageChannel> createKafkaConsumer(String name, final MessageChannel moduleInputChannel,
+														KafkaConsumerProperties properties, String group, long referencePoint) {
 
 		validateTopicName(name);
-
-		int numPartitions = accessor.getNumberOfKafkaPartitionsForConsumer();
-		Collection<Partition> allPartitions = ensureTopicCreated(name, numPartitions, defaultReplicationFactor);
+		int minKafkaPartitions = properties.getMinPartitionCount();
+		int instance = properties.getCount();
```

Comment:
I think `instanceCount` or at least `instances` would be more clear. Otherwise it seems like it's tied to a single instance (like `instanceIndex` is).